.travis.yml

```diff
@@ -5,3 +5,5 @@ rvm:
   - 2.2.*
 gemfile:
   - Gemfile
+before_install:
+  - gem update bundler
```

Comment:
Travis is failing tests in some version of ruby now.
https://travis-ci.org/sowawa/fluent-plugin-slack/builds/97396903
see also http://qiita.com/yuya_takeyama/items/446643b530b422e168c0

I will separate this patch to another PR.